src/actions/account.js

```diff
@@ -23,9 +26,15 @@ export const handleRedirectUrl = (previousLocation) => (dispatch, getState) => {
     }
 }
 
+const clearTempData = () => {
+    setTempAccount('')
+    setLinkdropData({})
```

Comment:
When is there a need to clear linkdrop data?

Reply:
When the user enters an accountId, and then exits the flow before completing recovery, we decided to flush the localStorage of their linkdrop data in case they switch users.

You should only ever be able to claim a linkdrop by clicking the link and completing the entire sign up process and claiming the linkdrop. If you leave the flow, your data should be cleared.

I don't believe we currently do that on unmount / window.unload, but we should.

Reply:
Handled by https://github.com/near/near-wallet/pull/700/commits/610672c372e1b5f94616bdbf709d0757fe213daf